x-pack/filebeat/input/httpjson/response.go

```diff
@@ -74,63 +74,68 @@ func newResponseProcessor(config *responseConfig, pagination *pagination, log *l
 	return rp
 }
 
-func (rp *responseProcessor) startProcessing(stdCtx context.Context, trCtx *transformContext, resp *http.Response) (<-chan maybeMsg, error) {
+func (rp *responseProcessor) startProcessing(stdCtx context.Context, trCtx *transformContext, resp []*http.Response) (<-chan maybeMsg, error) {
 	ch := make(chan maybeMsg)
```

Comment:
As there is a channel involved, could you please cover it with positive and negative tests?

Reply:
Sure, I will work on it.
Thanks